### DISCUSSION TITLE
web二步验证无法跨机使用

### DISCUSSION THREAD

**Discussion by DictXiong** [canokeys/canokey-product#16] [Q&A] [answered]
像github，cloudflare这样的二步骤验证，使用硬件秘钥，在一台机器上注册之后，在另一台机器上无法使用。

例如：在pc1上添加了canokey，并验证在pc1上能正常使用；随后到pc2上登陆同一个账户，就会提示

![origin_img_v2_94ac359b-4195-4269-8ad6-5228e1f2f61g](https://user-images.githubusercontent.com/41772157/178890647-de9d29cf-9539-48d2-82d7-fc9623d562ab.jpg)

随后，能在pc2上注册同一个canokey，然后同一个key就会对应多个：

![image](https://user-images.githubusercontent.com/41772157/178891274-529de360-859b-4041-86a4-2a7735b415f1.png)

这似乎是一个bug？期望表现应该是，在一台机器上注册key之后，能够在任意别的地方使用。

**Comment by dangfan** [comment, marked as answer]
应该是添加成了Windows Hello的密钥器，Windows弹出的第一个框需要点击取消，才是使用外置的。

**Reply by DictXiong** [reply]
确实如此……现在能用了，谢谢
